llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1005,7 +1005,7 @@ Value *VPInstruction::generate(VPTransformState &State) {
     if (getNumOperands() == 1) {
       Value *Mask = State.get(getOperand(0));
       return Builder.CreateCountTrailingZeroElems(Builder.getInt64Ty(), Mask,
-                                                  true, Name);
+                                                  /*ZeroIsPoison=*/false, Name);
```

Comment:
Could you extend the documentation for `FirstActiveLane` to state explicitly that any of the operands can be all zeros?